Userland/Libraries/LibWeb/HTML/CanvasRenderingContext2D.cpp

```diff
@@ -58,38 +58,59 @@ JS::NonnullGCPtr<HTMLCanvasElement> CanvasRenderingContext2D::canvas_for_binding
     return *m_element;
 }
 
+// 4.12.5.1.10 https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-fillrect
```

Comment:
Please omit the section number, which is for https://html.spec.whatwg.org/multipage/canvas.html#drawing-rectangles-to-the-bitmap